core/src/main/java/org/elasticsearch/index/mapper/ScaledFloatFieldMapper.java

```diff
@@ -268,8 +268,9 @@ public Query rangeQuery(Object lowerTerm, Object upperTerm, boolean includeLower
             return new FieldStats.Double(stats.getMaxDoc(), stats.getDocCount(),
                     stats.getSumDocFreq(), stats.getSumTotalTermFreq(),
                     stats.isSearchable(), stats.isAggregatable(),
-                    stats.getMinValue() == null ? null : stats.getMinValue() / scalingFactor,
-                    stats.getMaxValue() == null ? null : stats.getMaxValue() / scalingFactor);
+                    stats.getMinValue() == null ? 0.0 : stats.getMinValue() / scalingFactor,
```

Comment:
with regular numerics, we seem to be setting min/max to null in that case. Maybe we should look at the docFreq stat of the fieldstats and call the constructor that does not take min/max values when it is 0?